### PR TITLE
feature/notes

### PR DESCRIPTION
**Pull request by aylton-almeida**

### CODE REVIEW

File: lib/screens/categories/editCategory.screen.dart [hunk 1]
```diff
@@ -101,6 +101,17 @@ class _EditCategoryScreenState extends State<EditCategoryScreen> {
   }
 
   void _onDeletePress(int id) async {
+    Alerts.showAlertDialog(
+        context: context,
+        title: 'Delete category',
+        content: 'Are you shure you want to delete the category',
```
Comment (luccaromaniello) [inline lib/screens/categories/editCategory.screen.dart:107]: sure

File: lib/screens/categories/editCategory.screen.dart [hunk 2]
```diff
@@ -111,14 +122,16 @@ class _EditCategoryScreenState extends State<EditCategoryScreen> {
...
+          text: ErrorTranslator.categoryError(e),
+          color: Colors.red,
+        );
       else
         Alerts.showSnackBar(
-            context: context,
-            text: 'An error happened, please try again later',
-            color: Colors.red);
+          context: context,
+          text: 'An error happened, please try again later',
```
Comment (luccaromaniello) [inline lib/screens/categories/editCategory.screen.dart:132]: occurred no lugar de happened

File: lib/utils/errorTranslator.utils.dart [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 abstract class ErrorTranslator {
   static String authError(error) {
+    if (error.code == null)
+      return "An error happened, please try again later.";
```
Comment (luccaromaniello) [inline lib/utils/errorTranslator.utils.dart:6]: occurred

File: lib/utils/errorTranslator.utils.dart [hunk 4]
```diff
@@ -21,6 +23,8 @@ abstract class ErrorTranslator {
   }
 
   static String categoryError(error) {
+    if (error.code == null)
+      return "An error happened, please try again later.";
```
Comment (luccaromaniello) [inline lib/utils/errorTranslator.utils.dart:27]: occurred

File: lib/utils/errorTranslator.utils.dart [hunk 5]
```diff
@@ -32,6 +36,8 @@ abstract class ErrorTranslator {
   }
 
   static String noteError(error) {
+    if (error.code == null)
+      return "An error happened, please try again later.";
```
Comment (luccaromaniello) [inline lib/utils/errorTranslator.utils.dart:40]: occurred

**Review by luccaromaniello — APPROVED** [review]

**Review by luccaromaniello — APPROVED** [review]

File: lib/screens/notes/editNote.screen.dart [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+        print(e);
+        if (e is ServiceException && e.code != 'USER_NOT_CONNECTED')
+          Alerts.showSnackBar(
+              context: context,
+              text: ErrorTranslator.noteError(e),
+              color: Colors.red);
+        else
+          Alerts.showSnackBar(
+              context: context,
+              text: 'An error happened, please try again later',
```
Comment (luccaromaniello) [inline lib/screens/notes/editNote.screen.dart:65]: occurred

File: lib/screens/notes/editNote.screen.dart [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+      print(e);
+      if (e is ServiceException && e.code != 'USER_NOT_CONNECTED')
+        Alerts.showSnackBar(
+            context: context,
+            text: ErrorTranslator.noteError(e),
+            color: Colors.red);
+      else
+        Alerts.showSnackBar(
+            context: context,
+            text: 'An error happened, please try again later',
```
Comment (luccaromaniello) [inline lib/screens/notes/editNote.screen.dart:94]: occurred

**Review by luccaromaniello — APPROVED** [review]

**Review by luccaromaniello — APPROVED** [review]

File: lib/screens/categories/editCategory.screen.dart (hunk 1 above)
Comment (aylton-almeida) [reply lib/screens/categories/editCategory.screen.dart:107]: nu ate chorei

**Pull request merged**